### ISSUE TITLE
Add PULL_REQUEST_TEMPLATE.md

### ISSUE THREAD

**Issue by Wgil** [zurda/Github-Info#36]
Add template for pull new requests.

Example:
https://github.com/LLK/scratch-gui/blob/develop/.github/PULL_REQUEST_TEMPLATE.md

**Comment by zurda** [comment]
Thanks for suggesting this @Wgil . Excellent suggestion

**Comment by Wgil** [comment]
Long time no see

**Comment by zurda** [comment]
Yes, and apologies for the delay. I don't think I really started using github's notification until this year 😂 Only noticed this a few months ago. Sorry about that 🌹

**Comment by Wgil** [comment]
haha just saying hello! :)

Cheers
